src/VASL/build/module/map/boardPicker/Underlay.java

```diff
@@ -59,15 +61,17 @@ public void readData() {
 
   public Image loadImage() {
     Image underlayImage = null;
+    // Get the image from the module data archive...
     try(InputStream in = GameModule.getGameModule().getDataArchive().getInputStream("boardData/" + imageName)) {
       underlayImage = ImageIO.read(new MemoryCacheImageInputStream(in));
     }
     catch (IOException ex) {
     }
 
     if (underlayImage == null) {
-      try {
-        underlayImage = archive.getImage(imageName);
+      // ... or try to load it from the one passed in from the board
+      try (InputStream archiveIn = archive.getInputStream(imageName)) {
```

Comment:
I'm not convinced this is correct now, nor am I convinced this was correct before. The question to be answered first is: Where can underlay images be?

Reply:
You're right. Just looked through all the boards/src files for underlays - in every case, they only ever reference files in dist/boardData - so the second attempt appears to be defunct.

Reply:
Well, I'm not sure. Someone who is familiar with where the underlays are needs to comment.